docs/source/noise/index.rst

```diff
@@ -38,45 +36,55 @@ Currently, errors added in different columns are independent of each other.
 Available Noise Types
 ---------------------
 
-Pseudopeople applies the different types of row-based and column-based noise in
-the following order to mimic the data generation process by which a real dataset
-might be corrupted. The table lists all the available noise types, but not every
+The table lists all the available noise types, but not every
 type of noise will necessarily be applied to every dataset or every column. The
 configuration determines which noise types are actually used. The "Noise Type"
 column shows the name of the noise type in the configuration system.
 
-.. list-table:: Types of noise in order of application
-  :widths: 1 2 5 1
+.. list-table:: Types of row-based noise (``row_noise``)
+  :widths: 1 2 5 
   :header-rows: 1
 
   * - Noise Type
-    - Description
-    - Example
-    - Order
-  * - ``row_noise``
-    - **Row-based noise**
-    - ---
-    - ---
-  * - ``omit_row``
-    - Omit a row
+    - Config key 
+    - Example cause
+  * - Omit a row
+    - ``omit_row``
     - Neglecting to file a tax form on time
-    - 1
-  * - ``column_noise``
-    - **Column-based noise**
-    - ---
-    - ---
-  * - ``leave_blank``
-    - Leave a field blank
+
+.. list-table:: Types of column-based noise (``column_noise``)
+  :widths: 1 2 5 
+  :header-rows: 1
+
+  * - Noise Type
+    - Config key
+    - Example cause
+  * - Leave a field blank
+    - ``leave_blank``
     - Forgetting to write your name on the designated line
-    - 2
-  * - ``choose_wrong_option``
-    - Choose the wrong option from a fixed set of options
+  * - Choose the wrong option
+    - ``choose_wrong_option``
     - Marking the "Male" box when you meant "Female"
-    - 3
-
-.. todo::
+  * - Optical character recognition (OCR) error
+    - ``make_ocr_errors``
+    - Misreading an 'S' instead of a '5'
+  * - Make typos
+    - ``make_typos``
+    - Accidentally typing an 'l' instead of a 'k' because they are 
+      right next to each other on a QWERTY keyboard
+  * - Misreport age
+    - ``misreport_age``
+    - Reporting that you are 28 years old when you are actually 27
+  * - Write the wrong digits
+    - ``write_wrong_digits``
+    - Writing "2022" instead of "2002" in your date of birth
```

Comment:
In terms of real-world examples, it seems like this might fall more in the category of "typo" than randomly getting the digits wrong. Would a better example be getting a street number or unit number wrong, e.g., "732" instead of "932", or something that's more like just remembering the number wrong in a somewhat random way?

Reply:
Yeah, I see what you mean, though it can get a little tricky -- for example if you wrote 1997 instead of 1999 that couldn't be a typo (in our current approach) since 7 and 9 are not next to each other.

I like the idea of changing this example to be about street number though; I think that is a more believable example.

```suggestion
    - Writing "732 Main St" as your street address instead of "932 Main St"
```